### DISCUSSION TITLE
I have two ideas about content's front-matter

### DISCUSSION THREAD

**Discussion by gumingWu** [nuxt/content#2084] [Ideas]
- Add a new key like visible or display to control the article' visibility, like

```md
---
display: false
---
```

the article will be hidden on the menu

- Nuxt.mdc vscode extension may be able to show the front matter's meaning for user, they can quickly configure

**Comment by nobkd** [comment]
Meaning of the front matter: https://content.nuxtjs.org/guide/writing/markdown/#front-matter
The native parameters: https://content.nuxtjs.org/guide/writing/markdown/#native-parameters
There is a `draft` key to exclude pages from production.

The repository for the VS Code MDC extension is here: https://github.com/nuxtlabs/vscode-mdc
